var/spack/repos/builtin/packages/neic-finitefault/package.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class NeicFinitefault(PythonPackage):
+    """Wavelet and simulated Annealing SliP inversion (WASP).
+    This code uses a nonlinear simulated annealing inversion method to
+    model slip amplitude, rake, rupture time, and rise time on a discretized
+    fault plane, finding the solution that best fits the observations in
+    the wavelet domain."""
+
+    homepage = "https://code.usgs.gov/ghsc/neic/algorithms/neic-finitefault"
+
+    url = "https://code.usgs.gov/ghsc/neic/algorithms/neic-finitefault/-/archive/0.1.0/neic-finitefault-0.1.0.tar.gz"
+    git = "https://code.usgs.gov/ghsc/neic/algorithms/neic-finitefault"
+
+    maintainers("snehring")
+
+    license("CC0-1.0", checked_by="snehring")
+
+    version("20240410", commit="ef6a1a92d60549100885112e24a18e38d8d4ce0b")
+    version("0.1.0", sha256="36b400dfc418bf78a3099f6fc308681c87ae320e6d71c7d0e98a2738e72fb570")
+
+    resource(
+        name="fd_bank",
+        url="https://zenodo.org/records/7236739/files/fd_bank",
+        sha256="fe0f1a392cb9b6623c981de2a4fae405d9820b14e274e287e64731aede8ecb40",
+        expand=False,
+        when="@0.1.0:",
+    )
+    resource(
+        name="LITHO1.0.nc",
+        url="https://ds.iris.edu/files/products/emc/emc-files/LITHO1.0.nc",
+        sha256="4429bdf3fc2a5402064b40b059faf3a79d9ce0818feb1b13122e169af56f4b43",
+        expand=False,
+        when="@0.1.0:",
+    )
+    resource(
+        name="tectonicplates",
+        url="https://github.com/fraxen/tectonicplates/archive/339b0c56563c118307b1f4542703047f5f698fae.zip",
+        sha256="694ebf7090db07e47b07f1ae21175c4a5fa9c85bb79815680e439c1032407b95",
+        when="@0.1.0:",
+    )
+
+    depends_on("python@3.9:3.13", type=("build", "run"))
```

Comment:
Python versions are inclusive (https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges) so, from `pyproect.toml` for 20240410 and this line, I suspect you want:

```suggestion
    depends_on("python@3.9:3.12", type=("build", "run"))
```

Reply:
Legit typo on this one, whoops.